fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -181,6 +181,16 @@ ACTOR Future<Void> rocksDBMetricLogger(std::shared_ptr<rocksdb::Statistics> stat
 	}
 }
 
+Error statusToError(const rocksdb::Status& s) {
```

Comment:
Just curios, Is there any reason for not keeping this inside structRocksDBKeyValueStore?

Reply:
I needed to move it out of the `Writer` struct so that it would be visible inside the `Reader` struct. I moved it to namespace scope because I value referential transparency highly. Specifically, at namespace scope, the function has access to fewer variables/names/state/etc, so there is less that it could possibly do. It's much easier to reason about a function when it doesn't interact with anything outside of it, and a function at namespace scope is more likely to fulfill that property than a function at class/struct scope.